src/subcommand.rs

```diff
@@ -90,6 +93,8 @@ impl Subcommand {
         )?;
         let root_dir = manifest.parent().unwrap();
 
+        let parsed_manifest = Manifest::parse_from_toml(&manifest)?;
```

Comment:
We should probably put some more effort in reducing the `util` functions that do a lot of repeated directory walking and manifest parsing, or at least cache manifests that we've already parsed a couple times.

Reply:
maybe, don't think it's too bad currently